SportsHub/SportsHub.Api/Exceptions/CustomExceptionModels/BusinessLogicException.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Text.Json;
+
+namespace SportsHub.Api.Exceptions.CustomExceptionModels;
+
+public class BusinessLogicException: CustomException
```

Comment:
Exception related to business logic shouldn't be on API layer.

Reply:
Can u point me where to move it and also where i need to throw exceptions . I was thinking to throw them in repositories

Reply:
A business logic exception should be thrown on the business layer - here is a Service layer.
Exception class should be placed on the Domain level